Sources/MapboxCoreNavigation/RouteController.swift

```diff
@@ -261,13 +295,11 @@ open class RouteController: NSObject {
     }
     
     @objc func fallbackToOffline(_ notification: Notification) {
-        self.updateNavigator(with: self.routeProgress)
-        self.updateRouteLeg(to: self.routeProgress.legIndex)
+        updateNavigator(with: self.routeProgress, completion: { _ in })
```

Comment:
`updateNavigator` already updates routeLeg. I don't see a need to do this twice using two separate methods, given that both are async.